### PR TITLE
Field trip fixes July 30

### PR DESCRIPTION
**Pull request by landonreed**
This PR resolves a couple of field trip items:
- Fixes ibi-group/trimet-mod-otp#303
- Along with https://github.com/opentripplanner/otp-datastore/pull/7, fixes ibi-group/trimet-mod-otp#302

It also fixes a css issue with display of date time options in Firefox  by using flex (rendering is also improved with Chrome)
- Fixes ibi-group/trimet-mod-otp#304

### CODE REVIEW

File: lib/components/admin/field-trip-status-icon.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+import Icon from '../narrative/icon'
+
+const FieldTripStatusIcon = ({ status }) => {
```
Comment (binh-dam-ibigroup) [inline lib/components/admin/field-trip-status-icon.js:5]: The direct return `=> (` notation could be used here.

File: lib/components/admin/trip-status.js [hunk 2]
```diff
@@ -92,7 +90,7 @@ class TripStatus extends Component {
     return (
       <Full>
         <Header>
-          {this._renderStatusIcon()}
+          <FieldTripStatusIcon status={this._tripIsPlanned()} />
```
Comment (binh-dam-ibigroup) [inline lib/components/admin/trip-status.js:93]: The value passed to status is ambiguous here... are we passing a trip or a status? (see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_AND#using_and)

File: lib/components/form/call-taker/date-time-options.js [hunk 3]
```diff
@@ -190,40 +190,36 @@ export default class DateTimeOptions extends Component {
...
+            }}
+            value={timeInput || dateTime.format('H:mm')}
+          />
+        </OverlayTrigger>
         <input
           className='datetime-slim'
           onChange={this.handleDateChange}
           onKeyDown={onKeyDown}
           style={{
             border: 'none',
```
Comment (binh-dam-ibigroup) [inline lib/components/form/call-taker/date-time-options.js:218]: With the latest round of visual changes, could the date selector get a border too? (if so, make this control 2px wider for Firefox, and the `Plan` button needs a new height property)
image

**Review by binh-dam-ibigroup — APPROVED** [review]
Looks good with minor suggestions. Passing `_tripIsPlanned()` to the status prop seems ambiguous, maybe that needs to be clarified.

**Comment by landonreed** [comment]
Thanks @binh-dam-ibigroup. Comments addressed in a99744fd52aec3870de904faa616f4b42a5d90c8

**Review by evansiroky — APPROVED** [review]

**Comment by landonreed** [comment]
@binh-dam-ibigroup I changed it to `ok` in e8722a507e0ec9b959c571882c0adb7bc1d8d6be

**Pull request merged**